hasher-matcher-actioner/webapp/src/pages/ContentDetailsWithStepper.tsx

```diff
@@ -107,6 +107,18 @@ export default function ContentDetailsWithStepper(): JSX.Element {
               actionPerformedAt={actionPerformedAt}
             />
           )}
+
+          <hr />
```

Comment:
nit/unsure: feels like we should stick to Col/Row/Container for spacing etc, makes it easier to adjust style etc.

Reply:
Not sure what you mean. The HR is for visual separation. The actual spacing is being done with Col/Row/Container.

Reply:
I would have expected bootstrap-react to have something like <Divider /> but I guess not. This definitely works as is.